### PR TITLE
Small doc updates to `election-provider-multi-phase`

### PR DESCRIPTION
**Pull request by emostov**
This PR makes some trivial fixes to the `election-provider-multi-phase` module docs.

### CODE REVIEW

File: frame/election-provider-multi-phase/src/lib.rs [hunk 1]
```diff
@@ -158,11 +159,11 @@
...
 //! ## Future Plans
 //!
-//! **Challenge Phase**. We plan adding a third phase to the pallet, called the challenge phase.
-//! This is phase in which no further solutions are processed, and the current best solution might
+//! **Challenge Phase**. We plan on adding a third phase to the pallet, called the challenge phase.
+//! This is a phase in which no further solutions are processed, and the current best solution might
 //! be challenged by anyone (signed or unsigned). The main plan here is to enforce the solution to
 //! be PJR. Checking PJR on-chain is quite expensive, yet proving that a solution is **not** PJR is
-//! rather cheap. If a queued solution is challenged:
+//! rather cheap. If a queued solution is successfully challenged:
```
Comment (shawntabrizi) [inline frame/election-provider-multi-phase/src/lib.rs:166]: what does successfully mean here?

**Review by shawntabrizi — APPROVED** [review]

File: frame/election-provider-multi-phase/src/lib.rs (hunk 1 above)
Comment (emostov) [reply frame/election-provider-multi-phase/src/lib.rs]: Updated with https://github.com/paritytech/substrate/pull/9041/commits/3d7083574400a12c0e32de3aab67ed0cde7ae060

**Review by kianenigma — APPROVED** [review]

File: frame/election-provider-multi-phase/src/lib.rs [hunk 2]
```diff
@@ -158,27 +159,27 @@
...
 //! afterwards, and remove their solution (for a small cost of probably just transaction fees, or a
 //! portion of the bond).
 //!
 //! **Conditionally open unsigned phase**: Currently, the unsigned phase is always opened. This is
-//! useful because an honest validation will run our OCW code, which should be good enough to trump
-//! a mediocre or malicious signed submission (assuming in the absence of honest signed bots). If an
-//! when the signed submissions are checked against an absolute measure (e.g. PJR), then we can only
-//! open the unsigned phase in extreme conditions (i.e. "not good signed solution received") to
-//! spare some work in the validators
+//! useful because a honest validator will run substrate OCW code, which should be good enough to trump
```
Comment (coriolinus) [inline frame/election-provider-multi-phase/src/lib.rs:178]: I think `an honest` is correct here; we choose by the initial vowel sound, not the silent consonant.

**Review by coriolinus — APPROVED** [review]

File: frame/election-provider-multi-phase/src/lib.rs (hunk 2 above)
Comment (emostov) [reply frame/election-provider-multi-phase/src/lib.rs:178]: TIL :)

**Comment by emostov** [comment]
bot merge

**Pull request merged**